### PR TITLE
Add Lethe icon

### PR DESCRIPTION
**Pull request by ThePsionic**
lethe

### CODE REVIEW

**Comment by Wolvan** [comment]
I don't think gears is a good icon for lethe, personally

**Comment by ThePsionic** [comment]
@Wolvan Suggestions are always welcome

**Comment by hobinjk** [comment]
Lethe is also pretentiously/unhelpfully named, this would be a good time to improve both.

**Comment by ThePsionic** [comment]
@hobinjk I suggest the name "Speeder Upper" ;)

**Comment by Wolvan** [comment]
@ThePsionic No. Just No.

**Comment by nightpool** [comment]
What does lethe do? I was under the impression it was useless with the most
recent tumblr code

**Comment by ThePsionic** [comment]
As far as I am aware, Lethe clears out `.post_container`s that are high up on the dash to improve performance.

**Comment by ThePsionic** [comment]
Never mind, Lethe just clears out media posts. The cleared `.post_container`s seems to be Tumblr-native.

**Comment by nightpool** [comment]
Doesn't that not have any additional effect with native tumblr behavior
then?

**Comment by ThePsionic** [comment]
@nightpool I really don't know. I just now disabled Lethe on my low-end laptop and don't see any difference in performance.

**Comment by bvtsang** [comment]
I thought Lethe was named after [the mythological river](https://en.wikipedia.org/wiki/Lethe). Gears wouldn't make sense if that were true.

**Comment by ThePsionic** [comment]
TIL @bvtsang

**Comment by dlmarquis** [comment]
I think this is supposed to solve the problem where tumblr gets progressively more slow as you scroll the page. I don't use it, so it might not work, but the problem it supposedly fixes definitely exists.

**Comment by ThePsionic** [comment]
Okay, so I noticed it acually does something when you're really far down your dash. Without Lethe, Tumblr starts to lag like crazy.

@nightpool @Wolvan What do you suggest to use as icon instead?

**Comment by Wolvan** [comment]
Do you have any symbols you use to create the icons, @ThePsionic?

**Comment by ThePsionic** [comment]
@Wolvan I normally use [FontAwesome](https://fortawesome.github.io/Font-Awesome/icons/) for the icons.

**Comment by ThePsionic** [comment]
Does anyone have a better idea for the icon? Because I sure don't.

**Comment by bvtsang** [comment]
There are a couple of options:

1. Use something resembling a river or forgetfulness, since Lethe seems to be representing that.
2. As @hobinjk suggested, change the name of the extension to something different (e.g. "Forget Old Posts" or something else that I didn't make up in like 2 seconds), and then pick a different icon for that.

**Comment by Wolvan** [comment]
:+1: on the river idea

**Comment by dlmarquis** [comment]
I would suggest winged boots because it speeds tumblr up, but that's because I'm a bad person who can't resist the mythology theme.

**Comment by Wolvan** [comment]
As in Hermes' boots? I like that idea, works well with the mythology

**Comment by ThePsionic** [comment]
wingfoot
How 'bout that, @Wolvan and @dlmarquis?

**Comment by Wolvan** [comment]
I actually really like this

**Comment by bvtsang** [comment]
If you're going to use winged boots, then I suggest Lethe to be renamed as Hermes and its description mentioning speeding Tumblr up.

**Comment by Wolvan** [comment]
@bvtsang we already have a lot of extensions taht are named differently than the extension. (For example RDO or TT+), so this is kind of a moot point. But renaming it to hermes might be a good idea, we should definitely change the description though

**Comment by ThePsionic** [comment]
@Wolvan @bvtsang renamed as of your suggestion, and changed the description

**Comment by Wolvan** [comment]
Minor version bump, pls

**Comment by ThePsionic** [comment]
Done, @Wolvan

**Comment by Wolvan** [comment]
LGTM, but I want other people's opinions before merge

File: Extensions/lethe.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-//* TITLE Lethe **//
-//* VERSION 1.1.1 **//
-//* DESCRIPTION Forgets posts once they scroll off the screen. **//
-//* DEVELOPER hobinjk **//
+//* TITLE Hermes **//
+//* VERSION 1.2.0 **//
+//* DESCRIPTION Helps speed up your Tumblr experience **//
```
Comment (nightpool) [inline Extensions/lethe.js]: description is more vague then previous description :/

are we aiming to make lethe more of a "performance tweaks" extension? Are there other things we might want to migrate here eventually? (not in this PR, but soon)

would also appreciate a extension name change (cf. old stats, colorquotes) with notification.

**Comment by dlmarquis** [comment]
@nightpool @Wolvan I'm pro leaving the description as "helps speed tumblr experience", since it's better at telling users what the extension will do than the old one was, and putting future performance tweaks here. I'll give it a day or so for anyone to object before r+ing.

**Comment by dlmarquis** [comment]
@homu r+

The Week in Review comes out in a few hours, so as long as we post on time this week there shouldn't be much confusion.

**Pull request merged**